apps/web/src/app/(dashboard)/documents/[id]/edit-document.tsx

```diff
@@ -44,11 +47,15 @@ export const EditDocumentForm = ({
   fields,
   user: _user,
   dataUrl,
+  creatorEmail,
+  creatorName,
+  selfSignExist,
 }: EditDocumentFormProps) => {
   const { toast } = useToast();
   const router = useRouter();
 
   const [step, setStep] = useState<EditDocumentStep>('signers');
+  const [selfSign, setSelfSign] = useState<boolean>(selfSignExist);
```

Comment:
Then we can turn this into a memo that checks if our email appears in the recipients list

Reply:
noted...making a new commit.